rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-variable.adoc

```diff
@@ -0,0 +1,133 @@
+
+Editing or deleting a global variable affects all configurations that use the variable.
+
+Check the usage of a variable before changing or deleting it.
+
+If you want to store login data securely, use xref:processautomation-prepare-deployment-credential.adoc[credentials].
+
+== Before You Begin
+
+* Ask an administrator to xref:usermanagement-manage.adoc#assign-privileges-to-a-user[assign you the needed privileges]:
++
+[cols="1,2"]
+|===
+|*Action* |*Required Privileges*
+
+|Open the *Process Automation* module
+|Process Automation Open
+
+|Create and change global variables that can be used in the test phase
+|Global Variables Create for Test phase, Global Variables Edit for Test phase
+
+|Create and change global variables that can be used in the production phase
+|Global Variables Create for Production phase, Global Variables Edit for Production phase
+
+|Delete global variables
+|Global Variables Delete
+
+|===
+
+== Create a Global Variable
+
+Create a global variable that process team members can link to an activity parameter in an automation to overwrite the initial value of the activity parameter.
+
+To create a global variable:
+
+. Open the *Global Variables* view of the *Process Automation* module.
+. Click *Create*.
+. [[form-create-globalvariable]] Complete the *Create a New Global Variable* form:
+* *Phase affiliation*
++
+Phases in which the variable is permitted to be deployed. When you create or change the variable, your individual privileges determine which phases you can select:
++
+** *Test*
++
+The variable can be deployed in the test phase.
++
+** *Production*
++
+The variable can be deployed in the production phase.
+
+* *Type*
++
+Type of the global variable. The type of variable determines where it can be used. The original value defined in RPA Builder is typed. Select one of the following types:
++
+** *Integer*
++
+An integer number.
+** *Alphanumeric*
++
+A character string.
+** *Boolean*
++
+A logical value. Set the checkmark in the *Value* checkbox for the value `true` or remove the checkmark for the value `false`.
+** *Float*
++
+A floating-point number.
+** *Coordinate*
++
+Combination of two integer values for an X-Y coordinate.
+* *Value*
++
+You can only enter values of the specified type.
+
+== Check the Usage of a Global Variable
+
+Before editing a global variable, check its usage to avoid inadvertent side-effects:
+
+. Open the *Global Variables* view of the *Process Automation* module.
+. Click the *Usage* icon image:usage-icon.png[binoculars symbol,1.5%,1.5%] in the table row of the global variable you want to check.
+
+A window with a table shows you the configurations in which the global variable is used.
+
+==  Edit a Global Variable
+
+Edit a global variable to change its name or value. You cannot change the type of a global variable. Check the usage of the variable first to avoid inadvertent side-effects.
```

Comment:
```suggestion
Edit a global variable to change its name or value. You cannot change the type of a global variable. Check the usage of the variable first to avoid inadvertent side effects.
```